packages/trader/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -90,6 +99,12 @@ class AccountWizard extends React.Component {
         return this.state.step;
     }
 
+    getCountryCode = async () => {
+        await this.props.fetchResidenceList();
+        const { phone_idd } = this.props.residence_list.find((item => item.value === this.props.residence));
```

Comment:
doesn't `fetchResidenceList` return residence_list? we can store that and use it, instead of getting `residence_list` from client again, right?

Reply:
Yeah, I did not want to involve additional state when `residence_list` is already in the store. I could get and store it, but `fetchResidenceList` already does that in the store, so it retains reactivity, and I can pass the same object down, instead of creating it. 

So, the thought process behind this was not to get 2 different objects. the `this.props.fetchResidenceList` here is only invoked to make sure there is a `residence_list` in the store.